politeiawww/pi/process.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package pi
+
+import (
+	"context"
+
+	"github.com/decred/politeia/politeiad/plugins/pi"
+	v1 "github.com/decred/politeia/politeiawww/api/pi/v1"
+	"github.com/decred/politeia/politeiawww/user"
+)
+
+func (p *Pi) processSetBillingStatus(ctx context.Context, sbs v1.SetBillingStatus, u user.User) (*v1.SetBillingStatusReply, error) {
+	log.Tracef("processSetBillingStatus: %v", sbs.Token)
+
```

Comment:
The route settings should ensure only admins can access this route, but its not a bad idea to have a quick `if !u.Admin` sanity check here.  Sanity checks like this can help prevent bad things from happening in the event that some type of bug gets introduced that messes up the route settings.